cloud/disk_manager/test/acceptance/test_runner/lib/instance_policy.py

```diff
@@ -62,6 +64,11 @@ def obtain(self) -> Ycp.Instance:
                                          f' {self._zone_id}')
         try:
             yield self._instance
+        except Exception:
+            if self._do_not_delete_on_error:
+                _logger.info(f'Instance <id={self._instance.id}> will not be deleted')
```

Comment:
%s for logging formatting.

Reply:
It will be out of style with other logger usages